app/models/user.rb

```diff
@@ -1,2 +1,9 @@
 class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
+  has_many :likes
+
+  def self.last_three_posts(user_id)
+    Post.where(author_id: user_id).order(created_at: :desc).limit(3)
```

Comment:
- [ ] instead of writing `Post.where(author_id: user_id)` we can directly write `posts` as there is has_many relationship bw user & post.